examples/agentInteractionsDemo/index.html

```diff
@@ -0,0 +1,197 @@
+        <section class="conversation-list">
+            <div class="form-container">
+                <form @submit.prevent="filterCards">
+                    <div class="conversation-list-filter">
+                        <div class="date-picker">
+                            <div class="form-group row">
+                                <label for="starttime" class="col-2 col-form-label">From</label>
+                                <div class="col-10">
+                                    <input class="form-control" type="date" id="starttime" name="starttime">
+                                </div>
+                            </div>
+                            <div class="form-group row">
+                                <label for="endtime" class="col-2 col-form-label">To</label>
+                                <div class="col-10">
+                                    <input class="form-control" type="date" id="endtime" name="endtime">
+                                </div>
+                            </div>
+                        </div>
+                        <div class="form-group submit">
+                            <input type="submit" class="btn btn-outline-dark" value="Filter" id="filter">
+                        </div>
+                    </div>
+                </form>
+            </div>
+            <div class="conversations">
+                <div v-if="conv.id" v-for="conv in conversations"
+                    class="card">
+                    <div class="card-header">
+                        <span class="live-header" v-if="!conv.endTime">
+                            Live
+                        </span>
+                        <span class="evaluated-header" v-else-if="convEvalMap.has(conv.id)">
+                            Evaluated
+                        </span>
+                        <span class="finished-header" v-else>
+                            Finished
+                        </span>
+                    </div>
+                    <div class="card-body">
+                        <button class="btn btn-link">
+                            <h5>
+                                <a @click="viewInteraction(conv.id)">
+                                    {{conv.customer.name}}
+                                </a>
+                            </h5>
+                        </button>
+                        <p class="card-text mb-2 text-muted">
+                            Start: {{moment(conv.startTime).format('MMM Do YYYY, h:mm:ss A')}}
+                        </p>
+                        <p v-if="conv.endTime" class="card-text mb-2 text-muted">
+                            End: &nbsp {{moment(conv.endTime).format('MMM Do YYYY, h:mm:ss A')}}
+                        </p>
+                        <div class="dropdown dropright">
+                            <button v-if="convEvalMap.has(conv.id)"
+                                    id="dropdownMenuBtn"
```

Comment:
Do vue do anything to prepend context to ids inside components?  If not, these ids will techncially all have the same ids.  If you're not targeting them by id for a label, I would say remove the label.  css can target by class and you wont have duplicate ids.

Reply:
If this is used for automation, you'll have to ensure it's updated as well.